ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1961,6 +1961,61 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )altinfra-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build02
+    context: ci/prow/altinfra-periodics-images
```

Comment:
I think it's as if you are adding a new variant. And since you're defining an `images` stanza, it created this build. You'd have to define the job in the existing `__altinfra.yaml` config like https://github.com/openshift/release/pull/48788/files

Reply:
I moved these into the altinfra variant. Let's see how that works.